internal/signer/darwin/keychain/keychain.go

```diff
@@ -18,7 +18,7 @@
 package keychain
 
 /*
-#cgo CFLAGS: -mmacosx-version-min=10.12
+#cgo CFLAGS: -mmacosx-version-min=10.14
```

Comment:
Drop this change for now. It will be easier to rollback if it's in a separate commit / PR